packages/subproviders/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## v0.8.1 - _TBD_
+
+    * Introduce `JSONRPCRequestPayloadWithMethod` type (#465)
+    * Export `ErrorCallback` type. (#465)
+    * Make `handleRequest` private in `LedgerSubprovider` since it should only be used
```

Comment:
I don't agree. Private methods are methods that are used only by the class itself. If provider engine will be TS - this would fail.